code/ui/manager/src/components/sidebar/SearchResults.tsx

```diff
@@ -1,49 +1,67 @@
 import { styled } from '@storybook/theming';
-import { Icons } from '@storybook/components';
+import { Button, IconButton } from '@storybook/components';
 import { global } from '@storybook/global';
 import type { FC, MouseEventHandler, PropsWithChildren, ReactNode } from 'react';
 import React, { useCallback, useEffect } from 'react';
 import type { ControllerStateAndHelpers } from 'downshift';
 
 import { useStorybookApi } from '@storybook/manager-api';
 import { PRELOAD_ENTRIES } from '@storybook/core-events';
-import { ComponentNode, DocumentNode, Path, RootNode, StoryNode } from './TreeNode';
+import { transparentize } from 'polished';
+import { TrashIcon } from '@storybook/icons';
+import { TypeIcon } from './TreeNode';
 import type { Match, DownshiftItem, SearchResult } from './types';
-import { isCloseType, isClearType, isExpandType } from './types';
-// eslint-disable-next-line import/no-cycle
-import { getLink } from '../../utils/tree';
+import { isExpandType } from './types';
 import { matchesKeyCode, matchesModifiers } from '../../keybinding';
+// eslint-disable-next-line import/no-cycle
 import { statusMapping } from '../../utils/status';
+import { UseSymbol } from './IconSymbols';
 
 const { document } = global;
 
-const ResultsList = styled.ol({
+const ResultsList = styled.ul({
```

Comment:
why? i don't really care either way but i'd think that `ol` is more semantically aligned with search results?

Reply:
You're right. I can bring back `ol`. I don't think it really matters since we are not using visual elements.

Reply:
This is now back to using `ol`